src/libraries/System.Formats.Tar/System.Formats.Tar.sln

```diff
@@ -60,6 +66,7 @@ Global
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
+		{BE259E6E-B4F5-47DC-93D5-204297098A8C} = {6CF0D830-3EE9-44B1-B548-EA8750AD7B3E}
```

Comment:
Nice! Thanks. This is what I wanted: Nest the newly added StreamConformanceTests project under the tests folder.

Reply:
Yes, this is all auto-generated by the slngen tooling.